### ISSUE TITLE
(🐞) `dmypy inspect` 'No known type available for "NameExpr"'

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#14414]
```
👉 ${c:test.py} = "a = 1"
👉 dmypy start
Daemon started
👉 dmypy check test.py
Success: no issues found in 46 source files
👉 dmypy inspect test.py:1:1:1:1
No known type available for "NameExpr" (maybe unreachable or try --force-reload)
```

**Comment by ilevkivskyi** [comment]
https://mypy.readthedocs.io/en/stable/mypy_daemon.html
```
Important note: it is recommended to check files with --export-types since otherwise most inspections
will not work without --force-reload.
```

**Comment by KotlinIsland** [comment]
Seems like a prickly UX issue, maybe the error message could indicate that if no types are available at all.
